README.md

```diff
@@ -15,7 +15,138 @@ def deps do
 end
 ```
 
+## Configuration
+
+Provide your `TAG_ID` in your configuration file:
+
+```elixir
+# config/config.exs
+
+config :my_app, MyAppWeb.Gamco,
+  tag_id: "G-XXXXXXXXX",
+  active: false
+```
+
+And activate it in your `config/production.exs`
+
+```elixir
+# config/prod.exs
+
+config :my_app, MyAppWeb.Gamco, active: true
+```
+
+You can set up multiple `tag_id` in different environments if you need
+it. Just add the `tag_id` in the proper config file.
+
+## Usage
+
+Add a `MyAppWeb.Gamco` module as the following example:
+
+```elixir
+# lib/my_app_web/gamco.ex
+
+defmodule MyAppWeb.Gamco do
+  use Gamco, otp_app: :my_app
+end
+```
+
+Then import your Gamco module inside your phoenix view or component
+depending on what you are using. Example:
+
+```elixir
+# lib/my_app_web/views/layout_view.ex
+
+defmodule MyAppWeb.LayoutView do
+  import MyAppWeb.Gamco
+  ...
+end
+```
+
+Usually you'll want to add your Google Analytics script
+immediately after your `<head>` tag in your html template.
+
+```elixir
+# lib/my_app_web/templates/layout/root.html.heex
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <%= ga_javascript_tags() %>
+    ...
+  </head>
+  ...
+</html>
+```
+
+This will send the default `page_view` event every time the user
+navigates to a different path in your application. You can pass
+other dimensions if you want to collect extra data. Example:
+
+```elixir
+# lib/my_app_web/templates/layout/root.html.heex
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <%= ga_javascript_tags(extra_data: "FooBar") %>
+    ...
+  </head>
+  ...
+</html>
+```
+
+## Custom events
+
+Also, you can send custom messages to Google Analytics per page
+or when needed by using the `ga_tag` helper as the following example:
+
+```elixir
+# lib/my_app_web/templates/product/show.html.heex
+
+<%= ga_tag("event", "product_view", product_id: @product.id) %>
+```
+
+## Secure data
+
+There are sensitive information that you might not want to send
+it directly to Google Analytics to prevent some vulnerabilities.
```

Comment:
not want to send ~it~ directly ... remove it, not needed.

Reply:
Fixed.